src/tracer.js

```diff
@@ -29,6 +35,21 @@ export const ASYNC_HANDLER_REJECTED = 'async_handler_rejected';
 export const NON_ASYNC_HANDLER_ERRORED = 'non_async_errored';
 export const LEAK_MESSAGE =
   'Execution leak detected. More information is available in: https://docs.lumigo.io/docs/execution-leak-detected';
+const TIMEOUT_BUFFER_MS = 500;
+
+const setupTimeoutTimer = () => {
+  logger.debug('Timeout timer set-up started');
+  const { context } = TracerGlobals.getHandlerInputs();
+  const { remainingTimeInMillis } = getContextInfo(context);
+  if (TIMEOUT_BUFFER_MS < remainingTimeInMillis) {
+    GlobalTimer.setGlobalTimeout(async () => {
+      logger.debug('The tracer reached the end of the timeout timer');
```

Comment:
this message is confusing, how about saying something that tells the user what's actually happening? e.g. `Invocation is about to timeout, sending trace data back to Lumigo`

Reply:
👌